src/dpp/wsclient.cpp

```diff
@@ -43,7 +43,17 @@ websocket_client::websocket_client(const std::string &hostname, const std::strin
 	path(urlpath),
 	data_opcode(opcode)
 {
-	key = std::to_string(time(nullptr));
+	uint64_t k = (time(nullptr) * time(nullptr));
+	/* A 64 bit value as hex with leading zeroes is always 16 chars.
+	 *
+	 * The request MUST include a header field with the name
+	 * |Sec-WebSocket-Key|.  The value of this header field MUST be a
+	 * nonce consisting of a randomly selected 16-byte value that has
```

Comment:
But this doesn't make it 16-byte?

Reply:
as discussed to_hex will make it 16 characters by adding leading zeros up to 2x sizeof T, so for uint64_t, 16 chars